backend/src/main/java/com/bakdata/conquery/apiv1/QueryProcessor.java

```diff
@@ -377,4 +388,46 @@ public ExternalUploadResult uploadEntities(Subject subject, Dataset dataset, Ext
 				statistic.getUnreadableDate()
 		);
 	}
+
+	/**
+	 * Execute a {@link TableExportQuery} for a single Entity on some Connectors.
+	 *
+	 * @implNote we don't do anything special here, this request could also be made manually. We however want to encapsulate this behaviour to shield the frontend from knowing too much about the query engine.
+	 */
+	public Response getSingleEntityExport(Subject subject, String idKind, String entity, List<Connector> sources, String format, Dataset dataset, Range<LocalDate> dateRange) {
+
```

Comment:
Kannst du das bitte über eine QueryDescription lösen und nicht als extra Endpunkt

Reply:
Dann kann @Kadrian auf die selbe Abruflogik wie für normale Anfragen zurückgreifen

Reply:
Die Absicht dieses PRs war genau, dass es über einen separaten Endpunkt läuft, sonst könnte das Frontend ja auch einfach die Anfrage selber machen aber genau das wollen wir vermeiden.

Reply:
Aber das macht es ja jetzt auch, in dem Kai einen neuen Endpunkt ansprechen müsste. Anders müsste Kai nur einen SingleEntity Payload an den anderen Endpunkt senden. So wie bei den Formularen.

Reply:
Es wurde sich für einen extra Endpunkt entschlossen